crates/optimism/rpc/src/eth/transaction.rs

```diff
@@ -112,4 +136,10 @@ where
 
         Ok(OptimismTxMeta::new(Some(l1_block_info), l1_fee, l1_data_gas))
     }
+
+    /// Sets a `SequencerClient` for `eth_sendRawTransaction` to forward transactions to.
+    pub fn set_sequencer_client(&self, _sequencer_client: SequencerClient) {
+        // XXX FIXME YSG: This is a no-op for now.
+        // self.inner.1 = Some(sequencer_client);
+    }
```

Comment:
I see, on main the tx forwarder is wrapped in a `RwLock`
https://github.com/paradigmxyz/reth/blob/6459acb839ea70f491467a33c3f6a5b888dadfc5/crates/rpc/rpc/src/eth/core.rs#L205-L206
either wrap `RwLock<SequencerClient>`, or add a new method `RpcRegistryInner::eth_api_mut`. in general it's an anti pattern to return a mutable ref to a type built with builder pattern, but adding locks increases risk of creating live/deadlock bugs - however since lock was used on main already, should be fine here too.